### ISSUE TITLE
[[_TOC_]] does not behave as it is supposed to

### ISSUE THREAD

**Issue by lusamino** [SwissDataScienceCenter/renku-ui#2573]
## Motivation

For longer README, I use [[_TOC_]] in the README file to link to different sections. However, it does not seem to work in the Renku UI

![image](https://github.com/SwissDataScienceCenter/renku-ui/assets/8778401/cd9026b9-8459-49f8-86b3-3ef89c3435c0)

while, it does in GitLab

![image](https://github.com/SwissDataScienceCenter/renku-ui/assets/8778401/411ec362-540e-436b-96ae-857cc5a429ef)

## Proposal

It would ideal to have this solved, in order to get more consistent READMEs through platforms. 

Or, know how to implement this in a way that works in both Renku UI and GitLab (I tried already `[Cloning the repo, loading and using KG-FRUS](#1.-cloning-the-repo,-loading-and-using-kg-frus)` but it does not work seem to work.

Thank you!

**Comment by lorenzo-cavazzi** [comment]
Hello @lusamino , I gave this a look and adjusting the links might be enough. We have a more aggressive sanitize function to rename the links to match local files so that we don't redirect to GitLab accidentally (for reference, it's in the `RenkuMarkdownWithPathTranslation` component in "client/src/components/markdown/RenkuMarkdownWithPathTranslation.tsx"). This means that `#` breaks the link bringing the user to a 404 page. We can try to find a solution for this, it might not be too difficult.

The workaround might be simple tho: I tried to rewrite the link to `#1.-cloning-the-repo,-loading-and-using-kg-frus` to match the version with stripped extra chars, removing the `.` and the `,` and adding the name of the local file. The link becomes `README.md#1-cloning-the-repo-loading-and-using-kg-frus` and it seems to work fine in the RenkuLab file browser. Here is an example https://dev.renku.ch/projects/lorenzo.cavazzi.tech/project-3098-core-2-5-0rc3/files/blob/README.md

Would this solution be acceptable to you? Mind that clicking on the TOC from the readme file redirects you first to the file browser; for this I wouldn't have a workaround.
